ihatemoney/utils.py

```diff
@@ -47,6 +47,31 @@ def send_email(mail_message):
     return True
 
 
+def flash_email_error(error_message, category="danger"):
+    """Helper to flash a message for email errors. It will also show the
+    admin email as a contact if MAIL_DEFAULT_SENDER is set to not the
+    default value and SHOW_ADMIN_EMAIL is True.
+    """
+    admin_email = current_app.config.get("MAIL_DEFAULT_SENDER")
+    error_extension = "."
+    if (
+        admin_email
+        and admin_email[1] != "admin@example.com"
+        and current_app.config.get("SHOW_ADMIN_EMAIL")
+    ):
+        error_extension = " or contact the administrator at {}.".format(admin_email[1])
```

Comment:
```suggestion
    (admin_name, admin_email) = current_app.config.get("MAIL_DEFAULT_SENDER")
    error_extension = "."
    if (
        admin_email
        and admin_email != "admin@example.com"
        and current_app.config.get("SHOW_ADMIN_EMAIL")
    ):
        error_extension = f" or contact the administrator at {admin_email}."
```

f-strings are good, and we already requires Python >= 3.6
By the way, `MAIL_DEFAULT_SENDER` can't be `None`, so I destructured the assignment of it, to simplify the code.